pipeline_steps/maas.groovy

```diff
@@ -79,7 +79,7 @@ def verify() {
 void add_maas_env_vars(){
   List vars = get_maas_token_and_url()
   for (def i=0; i<vars.size(); i++){
```

Comment:
Don't know which defs are problematic but here's another that can be changed to an int.